SUM-262 SUM-264 SUM-266: fixes to the 4px white border

jenbreese opened the pull request:
# READY FOR REVIEW

# Summary
- Added a 4px white border bottom to the image banners
- Added a 4px white border bottom to the testimonial banners
- Added a 4px white border around the top grey header. 
- Added a 4px white border around the default grey image card for lg and up sizes. 
- Verified the lines line up when two are stacked with the same ratio.

# Review By (Date)
- when time allows

# Criticality
- Normal

# Urgency
- Normal

# Review Tasks

## Setup tasks and/or behavior to test

1. Check out this branch
2. On a page, add several stacked image banners and verify there is a 4px margin top between them.
3. On a page, add several stacked testimonial banners and verify there is a 4px margin top between them.
5. Verify the top grey header has a 4px white border around the whole thing.
6. On this page, `/students/undergraduate-students`, Verify the grey cards have a 4px white border.

### Site Configuration Sync

- Is there a config:export in this PR that changes the config sync directory?

## Front End Validation
- [ ] Design is approved by @ user?
- [ ] HTML validation: Is the markup using the appropriate semantic tags and [passes validation](https://validator.w3.org/nu/)? Or, [QA request ticket created](https://github.com/SU-SWS/template_warehouse/blob/master/jira_templates/QA_request_template.txt)?
- [ ] Cross-browser testing: Has been performed? Or, [QA request ticket created](https://github.com/SU-SWS/template_warehouse/blob/master/jira_templates/QA_request_template.txt)?
- [ ] Automated accessibility: Scans performed? Or, [QA request ticket created](https://github.com/SU-SWS/template_warehouse/blob/master/jira_templates/QA_request_template.txt)?
- [ ] Manual accessibility: Manually tested? Or, [QA request ticket created](https://github.com/SU-SWS/template_warehouse/blob/master/jira_templates/QA_request_template.txt)?

## Backend / Functional Validation
### Code
- [ ] Are the naming conventions following our standards?
- [ ] Does the code have sufficient inline comments?
- [ ] Is there anything in this code that would be hidden or hard to discover through the UI?
- [ ] Are there any [code smells](https://blog.codinghorror.com/code-smells/)?
- [ ] Are tests provided? eg (unit, behat, or codeception)

### Code security
- [ ] Are all [forms properly sanitized](https://www.drupal.org/docs/8/security/drupal-8-sanitizing-output)?
- [ ] Any obvious [security flaws or new areas for attack](https://www.drupal.org/docs/8/security)?

## General
- [ ] Is there anything included in this PR that is not related to the problem it is trying to solve?
- [ ] Is the approach to the problem appropriate?

# Affected Projects or Products
- Does this PR impact any particular projects, products, or modules?

# Associated Issues and/or People
[- SUM-262](https://stanfordits.atlassian.net/browse/SUM-262)
[- SUM-264](https://stanfordits.atlassian.net/browse/SUM-264)
[- SUM-265](https://stanfordits.atlassian.net/browse/SUM-265)
[- SUM-266](https://stanfordits.atlassian.net/browse/SUM-266)
[- SUM-267](https://stanfordits.atlassian.net/browse/SUM-267)
[- SUM-223](https://stanfordits.atlassian.net/browse/SUM-223)


# Resources
- [AMP Tool](https://stanford.levelaccess.net/index.php)
- [Accessibility Manual Test Script](https://docs.google.com/document/d/1ZXJ9RIUNXsS674ow9j3qJ2g1OAkCjmqMXl0Gs8XHEPQ/edit?usp=sharing)
- [HTML Validator](https://validator.w3.org/)
- [Browserstack](https://live.browserstack.com/dashboard) and link to [Browserstack Credentials](https://asconfluence.stanford.edu/confluence/display/SWS/External+Account+Credentials)

pookmish commented:
Is the border in the header correct? it feels out of place with the global message there
![image](https://github.com/user-attachments/assets/0abce297-2d0d-44fe-8ac3-eb1efea04fa3)

jenbreese commented:
Thanks for the catch. I'll check in with design.

jenbreese commented:
@pookmish Yes Design had envisioned something else. I'll rework this tomorrow.  
![image](https://github.com/user-attachments/assets/a3c55dc2-5a63-4117-99f2-9040f8da11c3)

jenbreese commented:
Ok. @pookmish I fixed the global message lines and tried it with and without the global message. I tested another line ticket and added it to the list of tickets.
